src/docker/DockerApi.ts

```diff
@@ -557,6 +621,7 @@ class DockerApi {
                     name: containerName,
                     Image: imageName,
                     Env: envs,
+                    Cmd: command,
```

Comment:
What would happen if `command` is undefined? Does it fail or does it execute the default `Cmd` defined in the image?